src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -225,4 +302,66 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
             this.props.onDeactivated(this.videoPlayer.current.video.video);
         }
     }
+
+    /**
+     * @name - Add Asset Timeline Tags
+     * @description - Draws small lines to show where visited and tagged frames are on
+     * the video line
+     * @member childAssets - Array of child assets in the video
+     * @member videoDuration - Length (in seconds) of the video
+     */
+    private addAssetTimelineTags(childAssets: any[], videoDuration: number) {
+        const innerHtml: string = this.getRenderedAssetTagLinesText(childAssets, videoDuration);
+        if (!this.timelineElement) {
+            // Find the progress holder so we can add the markers to that
+            const progressControls = document.getElementsByClassName("video-react-progress-control");
+            let progressHolderElement: Element = null;
+
+            // Find any that have an editor page content body type, this will be the one we use
+            Array.from(progressControls).forEach( (element) => {
+                let testingElement: Element = element;
+                while (testingElement) {
+                    if (testingElement.className === "editor-page-content-body") {
+                        progressHolderElement = element;
+                    }
+
+                    testingElement = testingElement.parentElement;
+                }
+            });
+
+            // If we found an element to hold the tags, add them to it
+            if (progressHolderElement) {
+                this.timelineElement = document.createElement("div");
+                this.timelineElement.className = "video-timeline-parent";
+                this.timelineElement.innerHTML = innerHtml;
+                progressHolderElement.appendChild(this.timelineElement);
+            }
+        } else {
+            this.timelineElement.innerHTML = innerHtml;
+        }
+    }
+
+    /**
+     * @name - Get Rendered Asset Tag Lines Text
+     * @description - Gets the HTML text for the rendered asset tag lines
+     * @member childAssets - Array of child assets in the video
+     * @member videoDuration - Length (in seconds) of the video
+     */
+    private getRenderedAssetTagLinesText(childAssets: any[], videoDuration: number): string {
+        const tagTimeLines: any = [];
+
+        // Add some markers for frames that have been visited with yellow and tagged with green
+        for (const childAsset of childAssets) {
+            // Calcualte the left position
+            const childPosition: number = (childAsset.timestamp / videoDuration);
+            tagTimeLines.push(<div key={childAsset.timestamp}
```

Comment:
We should add click handlers on the lines so we don't need to hook into the seek behavior.

Reply:
I was worried about having so many click handlers, but I can try this.

Reply:
Added